library/common/extensions/stat_sinks/metrics_service/metrics_service.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package envoymobile.extensions.StatSinks.MetricsService;
```

Comment:
```suggestion
package envoymobile.extensions.stat_sinks.metrics_service;
```

Reply:
Note that this means we will have to change this type in other places (including the service_method).